File: core-java-practice/gcr-codebase/programming-elements/PowerCalculation.java
import java.util.*;

public class PowerCalculation {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        //take input from user
        double base = sc.nextDouble();
        double ex = sc.nextDouble();
       double r =  Math.pow(base,ex);//cal power
        System.out.print(r);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/PowerCalculation.cs

class PowerCalculation {
    static void Main(String[] args) {
        //take input from user
        double baseval = Convert.ToDouble(Console.ReadLine());
        double ex = Convert.ToDouble(Console.ReadLine());
        double r =  Math.Pow(baseval,ex);//cal power
        Console.WriteLine(r);
    }
}
